src/extension.ts

```diff
@@ -55,12 +55,27 @@ async function init(context: ExtensionContext, disposables: Disposable[]): Promi
             return
         }
 
+        const getHostName = () => {
+            if ( os === 'win32' &&  absolutePath.slice(0,2) === '\\\\') {
+                let filePath = absolutePath.split('\\')
+                return _.compact(filePath)[0]
+            } else if ( os === 'darwin' && absolutePath.slice(0,9) === '/Volumes/') {
+                return 'openhabianpi.local'
+            } else {
+                return 'localhost'
+            }
+        }
+
+        const getFileName = () => {
+            return (os === 'win32') ? absolutePath.split('\\').pop() : absolutePath.split('/').pop()
```

Comment:
Just a thought. We could utilize [Node.js path library](http://devdocs.io/node/path#path_path_join_paths) so it does the detection for us :)